doc/api/url.md

```diff
@@ -129,7 +129,7 @@ return `true`.
 
 #### `new URL(input[, base])`
 
-<!--
+<!-- YAML
 changes:
```

Comment:
Should this one have `added`, too?

Reply:
I'd be nice to have for sure, but I'll let that to any git blame explorer motivated enough to find the correct info.